homeassistant/components/discovery/__init__.py

```diff
@@ -21,7 +21,7 @@
 from homeassistant.helpers.discovery import async_load_platform, async_discover
 import homeassistant.util.dt as dt_util
 
-REQUIREMENTS = ['netdisco==2.3.0']
+REQUIREMENTS = ['netdisco==2.4.0']
```

Comment:
Can we break out this to a separate PR?

Reply:
Sure will do.

Reply:
Done. Moved that to new PR https://github.com/home-assistant/home-assistant/pull/21914